<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>197</width>
      <height>211</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>ID:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="txtID">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Voltaje:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="txtVoltaje">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Posicion X:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="txtPx">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Posicion Y:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="txtPy">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Red:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="txtRed">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_8">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Green:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="txtGreen">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
       </property>
       <property name="text">
        <string>Blue:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="txtBlue">
       <property name="styleSheet">
        <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnAgregarFinal">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Agregar al final</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>250</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Cerrar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>CAPTURAR NEURONAS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAgregarInicio">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Agregar al inicio</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>10</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Lista:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>280</y>
      <width>75</width>
      <height>24</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Segoe UI&quot;;</string>
    </property>
    <property name="text">
     <string>Mostrar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>762</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
